### PR TITLE
New Blog Post

### PR DESCRIPTION
**Pull request by AnthonyMadia**
Blog post on Istanbul ETHPrivacy Hackathon.

### CODE REVIEW

File: packages/docs/blog/2023-05-15-unirep-in-turkey.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+<img src="static/img/../../../static/img/engin-yapici-WA1u0scVLZU-unsplash.jpg" alt="Alt Text" width="600" style="display: block; margin: 0 auto;">
+
+
+# UniRep Travels to Turkey
+
+In April, the [UniRep Protocol](https://developer.unirep.io/) was presented to a group of software engineers interested in privacy-related technology at the [ETHPrivacy](https://www.leadingprivacy.com/istanbul) conference for a privacy hackathon in Istanbul, Turkey.
+
+The location for the event was relevant as Turkey is one of the fastest-growing countries for the [adoption](https://www.reuters.com/technology/cryptoverse-digital-coins-lure-inflation-weary-argentines-turks-2023-05-02/#:~:text=Ownership) of blockchain-related products. More importantly, there are concerns within Turkey about freedom of speech and other human rights where privacy can help, so it made sense to have a privacy conference there.
```
[the remark this answers is not in the document]
Comment (AnthonyMadia) [reply packages/docs/blog/2023-05-15-unirep-in-turkey.md]: Okay -- updated in commit.

File: packages/docs/blog/2023-05-15-unirep-in-turkey.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+slug: unirep-travels-to-turkey-2023
+title: UniRep Travels to Turkey, 2023
+authors: [anthony]
+tags: [unirep in turkey]
+---
+
+<img src="static/img/../../../static/img/engin-yapici-WA1u0scVLZU-unsplash.jpg" alt="Alt Text" width="600" style="display: block; margin: 0 auto;">
```
Comment (vivianjeng) [inline packages/docs/blog/2023-05-15-unirep-in-turkey.md:8]: you missed a `/` before `>`
and I think the path could be `../static/img/engin-yapici-WA1u0scVLZU-unsplash.jpg`

File: packages/docs/blog/2023-05-15-unirep-in-turkey.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The term "reputation" in UniRep's name might be confusing at first. Some may first think about reputation being what a certain social group may think about them. In the context of UniRep, though, reputation is data. With this in mind, we can start to imagine what data can be associated with an anonymous developer that can build their "reputation" so they can push to main.
+
+To illustrate, let's assume you have specific criteria for selecting developers who can push to the main branch, such as requiring a certain number of stars on their projects, a minimum of 5 personal projects, and/or X amount of commits on a project on branches that are not the main branch. UniRep has the capacity to do all of that. Check out the hackathon project [here](https://github.com/gagichce/zk-dev), which was bootstrapped with [create-unirep-app](https://developer.unirep.io/docs/getting-started/create-unirep-app) (note: this will direct you to V2 beta 3 of the protocol). All the mentioned GitHub data can be linked to their anonymous identity, lending credibility to their anon identities.
```
Comment (vivianjeng) [inline packages/docs/blog/2023-05-15-unirep-in-turkey.md:23]: I think you don't need to specify the version of `create-unirep-app`
in the future, the url of the documentation will always be the latest published version

File: packages/docs/blog/2023-05-15-unirep-in-turkey.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Simplistically, what UniRep allows developers to do is _associate data_ with _anonymous users_; UniRep is acting as memory for zk-applications in this regard.
```
Comment (vivianjeng) [inline packages/docs/blog/2023-05-15-unirep-in-turkey.md:25]:
```suggestion
Simplistically, what UniRep allows developers to do is _associate data_ with _anonymous users_; UniRep is acting as storage for zk-applications in this regard.
```
I am not sure about this
maybe memory is also correct

**Review by vivianjeng — COMMENTED** [review]
Nice article 👏🏻

**Comment by AnthonyMadia** [comment]
Thanks. Will implement changes you mentioned. @vivianjeng

**Review by vimwitch — APPROVED** [review]

**Pull request merged**